Adding documentation for FM and COOP/COEP (#1)
salcho/struts-site: source/core-developers/interceptors.md

|[Annotation Workflow Interceptor](annotation-workflow-interceptor.html)|annotationWorkflow|Invokes any annotated methods on the action.|
|[Chaining Interceptor](chaining-interceptor.html)|chain|Makes the previous Action's properties available to the current Action. Commonly used together with <result type="chain"> (in the previous Action).|
|[Checckbox Interceptor](checkbox-interceptor.html)|checkbox|Adds automatic checkbox handling code that detect an unchecked checkbox and add it as a parameter with a default (usually 'false') value. Uses a specially named hidden field to detect unsubmitted checkboxes. The default unchecked value is overridable for non-boolean value'd checkboxes.|

> |[COEP Interceptor](coep-interceptor.html)|coep|Implements the COEP policy on incoming requests used to protect a document from loading any non-same-origin resources which don't explicitly grant the document permission to be loaded.|

Reviewer: ```suggestion
|[COEP Interceptor](coep-interceptor.html)|coep|Implements the Cross-Origin Embedder Policy on incoming requests used to protect a document from loading any non-same-origin resources which don't explicitly grant the document permission to be loaded.|
```

|[Conversion Error Interceptor](conversion-error-interceptor.html)|conversionError|Adds conversion errors from the ActionContext to the Action's field errors|
|[Cookie Interceptor](cookie-interceptor.html)|cookie|Inject cookie with a certain configurable name / value into action. (Since 2.0.7.)|
|[Cookie Provider Interceptor](cookie-provider-interceptor.html)|cookieProvider|Transfer cookies from action to response (Since 2.3.15.)|

> |[COOP Interceptor](coop-interceptor.html)|coop|Implements the COOP policy on incoming requests used to isolate resources against side-channel attacks and information leaks.|

Reviewer: ```suggestion
|[COOP Interceptor](coop-interceptor.html)|coop|Implements the Cross-Origin Opener Policy on incoming requests used to isolate resources against side-channel attacks and information leaks.|
```

|[Create Session Interceptor](create-session-interceptor.html)|createSession|Create an HttpSession automatically, useful with certain Interceptors that require a HttpSession to work properly (like the TokenInterceptor)|
|[Clear Session Interceptor](clear-session-interceptor.html)|clearSession|This interceptor clears the HttpSession.|
|[Debugging Interceptor](debugging-interceptor.html)|debugging|Provides several different debugging screens to provide insight into the data behind the page.|
|[Default Workflow Interceptor](default-workflow-interceptor.html)|workflow|Calls the validate method in your Action class. If Action errors are created then it returns the INPUT view.|
|[Exception Interceptor](exception-interceptor.html)|exception|Maps exceptions to a result.|
|[Execute and Wait Interceptor](execute-and-wait-interceptor.html)|execAndWait|Executes the Action in the background and then sends the user off to an intermediate waiting page.|

> |[Fetch Metadata Interceptor](fetch-metadata-interceptor.html)|fetchMetadata|Implements the Fetch Metadata policy on incoming requests used to protect against CSRF, XSSI, and cross-origin information leaks.|

Reviewer: ```suggestion
|[Fetch Metadata Interceptor](fetch-metadata-interceptor.html)|fetchMetadata|Implements Resource Isolation Policies on incoming requests used to protect against CSRF, XSSI, and cross-origin information leaks.|
```
